iceoryx_posh/include/iceoryx_posh/internal/roudi/roudi.hpp

```diff
@@ -41,9 +41,9 @@ using namespace iox::units::duration_literals;
 class RouDi
 {
   public:
-    // indicate whether the message queue thread will start directly or deferred
+    // indicate whether the IPC channel thread will start directly or deferred
     // this is important for derived classes which may need to initialize their members before the thread starts
-    enum class MQThreadStart
+    enum class RuntimeMessagesThreadStart
```

Comment:
the documentation talks about `IPC channel threads` ... oh, and please make it `/// @brief` :D

Reply:
:laughing: Fixed